pulseapi/profiles/test_views.py

```diff
@@ -504,3 +504,20 @@ def test_profile_categories(self):
         self.assertListEqual(categories['profile_types'], list(ProfileType.objects.values_list('value', flat=True)))
         self.assertListEqual(categories['program_types'], list(ProgramType.objects.values_list('value', flat=True)))
         self.assertListEqual(categories['program_years'], list(ProgramYear.objects.values_list('value', flat=True)))
+
+    def test_profile_view_is_active_filter(self):
```

Comment:
**Suggestion** Similar to the other tests, it would be good to also create a sanity check object that is `is_active=True` and is included in the response.

Reply:
Updated!